napari/conftest.py

```diff
@@ -0,0 +1,59 @@
+from typing import List
+import warnings
+
+import pytest
+from napari import Viewer
+from qtpy.QtWidgets import QApplication
+
+
+def pytest_addoption(parser):
+    """An option to show viewers during tests. (Hidden by default).
+
+    Showing viewers decreases tests by about %18.  Note, due to the placement
```

Comment:
What does this mean?  Decreases test speed?

Reply:
whoops.  yeah, fixed.